app/Commands/Self/UpdateCheck.php

```diff
@@ -14,7 +14,7 @@
  */
 class UpdateCheck extends Command {
 
-    public const TIME_BETWEEN_CHECKS = '7 days';
+    public const TIME_BETWEEN_CHECKS = '2 days';
```

Comment:
Awesome, TIL `strtotime()` is for parsing natural language, not just converting dates/times. 🎉